### ISSUE TITLE
Constant REDIS_VERSION does not reflect actual version, at least for RC series

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ashish21590/redis#462]
```
REDIS_VERSION, defined in src/version.h, does not reflect the actual version, 
at least for the RC series.  In particular, REDIS_VERSION in 
redis-2.2.0-rc4.tar.gz is set to 2.1.12.

This means that "redis-server --version" reports its version as 2.1.12, which, 
among other things, makes it difficult for automated build scripts (e.g. chef) 
to determine what version of redis is really installed, and therefore whether 
it should be upgraded.

I suggest that REDIS_VERSION should always match the the version number given 
in the filename; in the case of redis-2.2.0-rc4.tar.gz REDIS_VERSION should be 
"2.2.0-rc4".
```

Original issue reported on code.google.com by `ithinkihaveacat` on 20 Feb 2011 at 11:48

**Comment by GoogleCodeExporter** [comment]
```
This is fixed for 2.2.0, but it would be helpful if REDIS_VERSION reflected the 
actual version in future.
```

Original comment by `ithinkihaveacat` on 22 Feb 2011 at 10:39

**Comment by GoogleCodeExporter** [comment]
```
The current version numbering scheme uses odd minor versions for non-stable 
releases. Because of this, non-stable 2.2 releases use 2.1. The big advantage 
of doing this is that you always know x.y.z is a stable release where y is an 
even number. This would no longer hold when adding in a suffix (for 
rc/pre/alpha/etc.). I agree with you that it would be easier on the eyes to use 
a suffix for RCs, but it is harder for version management since you always need 
to check for a suffix to determine if it is a stable release or not.
```

Original comment by `pcnoordh...@gmail.com` on 6 Mar 2011 at 5:34
- Changed state: **WontFix**
